### DISCUSSION TITLE
epyc.q におけるtype1a と type1bの指定について

### DISCUSSION THREAD

**Discussion by manabuishii** [nig-sc/nigsc_homepage2#279] [Q&A] [answered]
お世話になっております。
石井と申します。

epyc.qにある、type1aとtype1bは明示的に指定することは可能でしょうか？

1CPUコア4GB程度を使用しているプログラムで、見ている限りでは、type1aだと動くのですが、type1bにジョブが入ると実行が失敗するような気がしています。

また、
マシン名や、qacct などから、type1aやtype1bどちらにはいったかを知ることは可能でしょうか？

よろしくおねがいします。

参考にしたページ
https://sc.ddbj.nig.ac.jp/guides/hardware

**Comment by sc-fs** [comment, marked as answer]
現在（2023年2月9日時点）、一般解析区画にて定義しているepyc.qにある計算ノード（25台）は
Type 1bのみで構成されています。
そのため、Type 1bにしか、ジョブは投入できません。

Thin計算ノードのType 1a　やType 1b　はノード名で判別できます。

  Thin計算ノード Type 1a(Naples) ->  at001 ～　at136
  Thin計算ノード Type 1b(Rome)   ->  at137 ～　at164
